buildSrc/src/main/groovy/csde.gradle

```diff
@@ -148,6 +148,18 @@ tasks.register("stopCorda") {
     }
 }
 
+tasks.register("stopAndCleanCorda") {
+    group = cordaGroup
+    dependsOn('clean')
+    doLast {
+        try {
+            cordaLifeCycle.stopCorda()
+        } catch (Exception ignored) {
```

Comment:
I think it's OK to ignore ... but at least we should log it stating it's ignored.
Hiding exceptions completely may confuse later users